languages/csharp/exercises/concept/nullability/.docs/instructions.md

```diff
@@ -0,0 +1,30 @@
+In this exercise you'll be writing code to helping printing name
+badges for factory employees.
+
+Employees have an id, a name and a department name. Badge labels are
+formatted as follows: `"[id] - [name] - [DEPARTMENT]"`
```

Comment:
I like how you've managed to make it clear that the department is in uppercase! I wonder if we should also capitalize the name itself, which would seem to make sense for the badge. Then people can also see the difference between calling a method on a nullable and non-nullable strings and are probably even more likely to make the null-reference mistake :) What do you think?

Reply:
I started doing that... but 

1. we already need to call a method on a non nullable string on the `PrintLabel` method...
2. if all strings are upper case, then one can just make the whole string upper case, it's easier to explain, and easier to implement

Reply:
Oh yes, excellent arguments! Let's leave it as is.